src/FakePAB/CardanoCLI.hs

```diff
@@ -78,13 +79,32 @@ submitScript config UnbalancedTx {unBalancedTxTx, unBalancedTxUtxoIndex} = do
       signTx preparedTx config.signingKeyFile
       txId <- getCLITxId preparedTx
 
-      if config.dryRun
+      if config.live
         then do
-          putStrLn $ "Dry run, not submitting transaction: " <> show txId
-          pure $ Right txId
-        else do
-          putStrLn $ "Submitting transaction: " <> show txId
-          fmap (txId <$) $ submitTx preparedTx config
+          confirm <- confirmTxSubmission
+          if confirm
+            then do
+              putStrLn $ "Submitting transaction: " <> show txId
+              fmap (txId <$) $ submitTx preparedTx config
+            else printDryrun txId
+        else printDryrun txId
+  where
+    confirmTxSubmission :: IO Bool
+    confirmTxSubmission = do
+      putStr "Are you sure you want to submit this transaction? [yes/no] "
```

Comment:
Oh! Seems theres been a minor minunderstanding - the confirmation should be at the start of the script, not for each transaction.
Otherwise, for airdrops of like 30 transactions, we'd need to confirm 30 times